packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -418,11 +418,11 @@ Example configuration:
 
 [source,yml]
 ------------------------------------------------------------------------------
-protocols:
-  http:
-    ports: [80, 8080]
-    send_response: true
-    include_body_for: ["text/html"]
+  protocols:
```

Comment:
@tsg Thinking if I should also do here packetbeat.protocols instead of changing the indentation. At the same time it "only" describes the protocols part. Same for all other doc parts.